.vscode/c_cpp_properties.json.default

```diff
@@ -0,0 +1,20 @@
+{
+    "configurations": [
+        {
+            "name": "stm8",
+            "includePath": [
+                "./src/controller",
+                "./src/common",
+                "./src/common/STM8S_StdPeriph_Lib/inc"
+            ],
+            "defines": [
+                "STM8S105",
+                "__interrupt(SDCC_IntelliSense_Hack)="
+            ],
+            "cStandard": "c99",
+            "intelliSenseMode": "gcc-arm",
+            "compilerPath": "C:/msys64/usr/bin/gcc.exe"
```

Comment:
`c_cpp_properties.json.default` can be renamed to `c_cpp_properties.json` and `compilerPath` modified.   It's annoying that this line is necessary and has to be an absolute path.